database/migrations/2022_02_18_023520_create_students_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('students', function (Blueprint $table) {
+            $table->id('id');
+            $table->string('name',50);
+            $table->integer('age');
+            $table->string('gender',10);
+            $table->text('address');
```

Comment:
```suggestion
            $table->id('id');
            $table->string('name', 50);
            $table->integer('age');
            $table->string('gender', 10);
            $table->text('address');
```
fix tương tự các chỗ khác